graphdatascience/query_runner/neo4j_query_runner.py

```diff
@@ -45,7 +51,7 @@ def run_query(
 
         self._verify_connectivity()
 
-        with self._driver.session(database=database) as session:
+        with self._driver.session(database=database, bookmarks=self._bookmarks) as session:
```

Comment:
I still think we might need to propagate the bookmarks to the server. But we should verify if the outer read transaction is enough. 

As the inner GDS writes will also need to know about the initial cypher writes

Reply:
or f.i. the native projection uses a new transaction to check if there is a label index.
Or a old cypher projection also uses internal transactions

Reply:
Is it not enough to stall the entire query on a "client" level for that? Is Nathan also propagating bookmarks to the server? Would love to discuss this